vignettes/how-to-update-released-site.Rmd

```diff
@@ -91,7 +117,7 @@ This directs pkgdown to "generate different sites for the development and releas
 
 The readr package demonstrates what happens in automatic development mode:
 
-[readr.tidyverse.org](https://readr.tidyverse.org) documents the released version, i.e. what `install.packages()` would deliver.  
+[readr.tidyverse.org](https://readr.tidyverse.org) documents the released version, i.e. what `install.packages()` would deliver.\
```

Comment:
Is this trailing slash intentional?
I see more below and some other whitespace changes that seem a bit odd.

Reply:
Appears to be visual editor changes which I won't bother to explore further.